src/test/objectstore/test_bluefs.cc

```diff
@@ -96,7 +96,7 @@ TEST(BlueFS, write_read) {
 }
 
 TEST(BlueFS, small_appends) {
-  uint64_t size = 1048476 * 128;
+  uint64_t size = 1ULL << 30;
   string fn = get_temp_bdev(size);
   BlueFS fs;
   ASSERT_EQ(0, fs.add_block_device(BlueFS::BDEV_DB, fn));
```

Comment:
I think with last change where we allow lesser than a zone size as input to allocator. But might hit an assert in BitAllocator.cc

1249   if (total_blocks < zone_size_block) {
   1250     debug_assert(0);
   1251   }


so if you remove assert, it should work with lesser number of blocks.

worth trying before making changes to unit test cases.